components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/SAMLLogEventConstants.java

```diff
@@ -0,0 +1,43 @@
+package org.wso2.carbon.identity.sso.saml;
+/*
+ * Copyright (c) 2023, WSO2 Inc. (https://www.wso2.org) All Rights Reserved.
```

Comment:
update the license header

Reply:
Removed this class and moved to string constant. Using enum might give some additional restrictions. In future, we need to have an osgi service to register all the suppported events. Using that, we can have some REST API to get the supported events